src/optionsMenu/MicTab.tsx

```diff
@@ -124,7 +156,7 @@ export function MicTab() {
             height={40}
             source="mic-precomp"
             range={[-70, 0]}
-            greenRange={[-14, -10]}
+            greenRange={state.processing ? [-14, -10] : [-10, -5]}
```

Comment:
While we're here, should we maybe expand this range a little? It's very narrow, which leads to people ignoring it.